script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_1.txt

```diff
@@ -30,40 +30,40 @@ there is nothing to be gained by seeing such things-----
 <Page8>
 "------hmnnn."
 I wake up from my old memories.
-I feel the morning light through my eyelids and crawl my hands under my pillow with my eyes closed.
+I feel the morning light through my eyelids and feel around on beside my pillow with my eyes closed.
 <Page9>
 ...Huh...where are my glasses...?
-Feeling uneasy, I grope around the right side of the pillow to try and find my glasses, and quickly pick them up.
+Feeling uneasy, I grope around beside my pillow on the right, just in case, trying to find my glasses. I quickly find them.
```

Comment:
`grope around beside my pillow on the right` feels a bit awkward - might keep it as `grope around the right hand side of my pillow`, or with liberties `grope around on the bed to the right of the pillow` if it were me